bot/repository.py

```diff
@@ -59,3 +63,18 @@ def create_bot(self, bot_create: BotCreate):
                 new_bot = BotModel(**bot_create.model_dump(), id=id)
                 session.add(new_bot)
                 session.commit()
+
+    def update_bot(self, bot_id, bot_update: BotUpdate = None):
+        with self.create_session() as session:
+            with self.logger.catch(message="update bot error", reraise=True):
+                bot = session.query(BotModel).filter(BotModel.id == bot_id).first()

```

Comment:
I think the part where we get the bot using its ID can be moved into a separate function because the get bot by ID api will be needed to send questions to the respective bot.

Reply:
Yes, thanks for the reminder. I'll do that